packages/api-gateway/src/drivers/http/routes/files/index.js

```diff
@@ -0,0 +1,53 @@
+const multer = require('fastify-multer'); // or import multer from 'fastify-multer'
+const boom = require('@hapi/boom');
+const { filesAdapter } = require('../../adapters');
+const { TIPO_EXTENSION } = require('../../utils/constants');
```

Comment:
I think we were using camel case, maybe you put it because is a constant, but if that's the case we should update all constants to use this nomenclature.